test/hotspot/jtreg/compiler/rangechecks/TestRangeCheckHoistingScaledIV.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2022, 2023, Arm Limited. All rights reserved.
+ * Copyright (c) 2022, 2025, Arm Limited. All rights reserved.
```

Comment:
Leave that copyright untouched but add a Rivos line instead.

Reply:
Thanks! It's updated.